src/index.js

```diff
@@ -490,25 +490,45 @@ export default class HyperScreen extends React.Component {
   constructor(props) {
     super(props);
 
+    this.onUpdate = this.onUpdate.bind(this);
```

Comment:
FWIW, because we're using arrow functions in this components, we should be able to remove these bind operations

Reply:
good point, I can clean that up in a follow-up PR.